tests/pytests/test_json.py

```diff
@@ -1117,3 +1117,31 @@ def testRedisCommands(env):
         time.sleep(0.1)
         env.expect('JSON.GET', 'doc:1', '$').equal(None)
         env.expect('ft.search', 'idx', 'ri*', 'NOCONTENT').equal([0])
+
+@no_msan
+def testUpperLower(env):
```

Comment:
No need to get an env - we create a new one in the test
```suggestion
def testUpperLower():
```